tests/helpers.py

```diff
@@ -1,12 +1,32 @@
 import random
 
 
-def gen_random_object():
-    return {"id": random.randrange(1000000)}
+def gen_collection(object_generator_func, start_element=0, count=None,
+                   object_type="campaigns"):
+    if count is None:
+        random.randrange(10000)
+    result = []
+    i = 0
+    volume = count - start_element
+    for i in range(volume // 100):
+        page = gen_page(object_generator_func, count=count,
+                        object_type=object_type,
+                        start_element=start_element + i * 100)
+        result.append(page)
+    if volume % 100 != 0:
+        i = i + 1 if len(result) else 0
+        page = gen_page(object_generator_func, count=count,
+                        object_type=object_type,
+                        start_element=start_element + i * 100,
+                        num_elements=volume % 100)
+        result.append(page)
+    return result
```

Comment:
Can you keep `gen_collection` under `gen_page` so that we can see the actual diff? FYI, the functions are ordered that way because we usually order the functions "à la C", i.e. with functions used in other functions at the top, although this is not a strict convention nor something important.

Reply:
Done